session-signatures/getLitActionSessionSigs/src/index.ts

```diff
@@ -8,6 +8,8 @@ import {
 } from "@lit-protocol/auth-helpers";
 import * as ethers from "ethers";
 import { LocalStorage } from "node-localstorage";
+// @ts-ignore
```

Comment:
What was the error that needed to be ignored?

Reply:
Could not find a declaration file for module 'ipfs-only-hash'.

Reply:
Looks like we can use https://www.npmjs.com/package/typestub-ipfs-only-hash